README.md

```diff
@@ -1 +1,72 @@
-# judgement-day
\ No newline at end of file
+#judgement-day
+***
+## Feel Free to Rate FACN2 Mentors
+***
+###How to use our Website:
+To check out the app, Please clone this Repository.
+```javascript
+git clone
+```
+Install all the Node dependencies
+```javascript
+
+```
+Run start
+```javascript
+
+```
+---
+##Our Product:
+We want to build a web application so the students of FACN2 can rate the mentors on categories and
+### Users Stories:
+* Rate Mentors on Categories: eg. Hair
+* See percentage of how much the mentor got rating for one Category.
+* See Questions so you can vote based on the category.
+* Click on the Mentors Pictures to choose the one who is better according to the question.
+* There are many fun Categories you can rate mentors.
+
+### Stretch Goals:
+* enabling score table for voting.
+* choose Category to vote on for the mentor.
+* adding a timer for every question.
+---
+## Schema:
+#####*Mentors Table*
+|id|name|imageUrl
+| -|-|-
+|1|Mario|Mario's picture|
+|2|Shireen| Shireen's picture|
+
+#####*Questions Table*
+|id |category|question|
+|-|-|-|
+|1|hair|who's hair is better?|
+|2|singing|who's singing is better?|
+#####*Category Table*
+|id|mentor_id|rating|Shireen|Mario|
+|-|-|-|-|-|
+|1|2|14|4|0|
+|2|1|10|0|5
+---
+## Wirframe:
+![](https://user-images.githubusercontent.com/26799467/29571643-68418e36-8762-11e7-98dd-aa36da535bac.png)
+
+---
+
+## Code guidelines:
+* CamelCase for variable names
```

Comment:
We only just decided this but we could add that CSS classes will be written in snake_case